.github/workflows/backup-kubernetes-databases-inhouse.yml

```
@@ -0,0 +1,168 @@
+        run: kubectl config set-credentials default --token=${{ secrets.service-account }}
+
+      - name: Kubernetes set server with certificate account token
+        run: kubectl config set-cluster ${{ inputs.cluster }}  --insecure-skip-tls-verify --server=${{ inputs.url }}
+
+      - name: Kubernetes set context cluster
+        run: kubectl config set-context ${{ inputs.cluster }} --cluster=${{ inputs.cluster }} --user=default
+
+      - name: Kubernetes use context
+        run: kubectl config use-context ${{ inputs.cluster }}
+      - name: Run database backup
+        run: |    
+          backup="${{ inputs.backup }}"
+          
+          if [[ $backup == "shared-wordpress" ]]; then
+            echo "We are going to backup shared wordpress databases."
+            select=$(kubectl get pod -n ${{ inputs.namespace }} --insecure-skip-tls-verify -l app.kubernetes.io/name=wordpress -o name)
+          else 
+            echo "We are going to backup postgres databases."
+            select=$(kubectl get pod -n ${{ inputs.namespace }} --insecure-skip-tls-verify -l service=database -o name)
+          fi
+          
+          mkdir "backup"
+          
+          for pod in ${select};
+          do
+            tmp=${pod##*/}
+            
+            if [[ $backup == "shared-wordpress" ]]; then
+              service=${tmp//-wordpress-*}
+            else
+              service=${tmp//-database-0}
+            fi
+            echo "Service: ${service}"
+
+            # no postgres db in clamav-cvdmirror
+            if [ "$service" = "clamav-cvdmirror" ]; then
+              echo "  skipping - no postgress db..."
+              continue
+            fi
+          
+            echo -n "  checking if pod is running..."
+            date=$(kubectl exec -n ${{ inputs.namespace }} --insecure-skip-tls-verify ${pod} --request-timeout=5s -- date) || true
+            if [[ $date == "" ]]; then
+              echo "  NOK - skipping...";
+              continue
+            fi
+            echo " OK";
+          
+            folder="backup/${{ inputs.cluster }}/${{ inputs.namespace }}/${service}"
+            echo -n "  creating export folder ${folder}...";
+            mkdir -p $folder
+            echo " OK"
+            
+            date=$(date '+date-%Y-%m-%d-time-%H-%M-%S')
+            
+            if [[ $backup == "shared-wordpress" ]]; then
+              echo -n "  obtaining service database pass...";
+              database_pass=$(kubectl exec -n ${{ inputs.namespace }} -c wordpress --insecure-skip-tls-verify ${pod} -- sh -c 'echo $WORDPRESS_DB_PASSWORD')
+              database=$service
+              echo " OK"
+          
+              echo -n "  exporting database from mariadb...";
+              kubectl exec -c mariadb -n ${{ inputs.namespace }} --insecure-skip-tls-verify mariadb-0 -- sh -c 'mysqldump --user=${service} --password=${database_pass} --all-databases' > "${folder}/${database}-${date}.sql" || true
```

```suggestion
              kubectl exec --container mariadb --namespace "${{ inputs.namespace }}" --insecure-skip-tls-verify mariadb-0 -- sh -c 'mysqldump --user="${service}" --password="${database_pass}" --all-databases' > "${folder}/${database}-${date}.sql" || true
```

Also, why `|| true`?